apps/studio/components/ui/Loading/Loading.tsx

```diff
@@ -1,12 +1,3 @@
-import Lottie from 'lottie-light-react'
-import loadingAnim from './Loading.anim.json'
+import Loading from 'ui/src/components/LogoLoader'
 
-const Connecting = () => (
-  <div className="w-full h-full flex flex-col items-center justify-center">
-    <div className="w-28">
-      <Lottie loop={true} autoplay={true} animationData={loadingAnim} />
-    </div>
-  </div>
-)
-
-export default Connecting
+export default Loading
```

Comment:
Should we get rid of this component, considering it does nothing but re-export LogoLoader?